dhcpv6/dhcpv6_test.go

```diff
@@ -121,23 +69,17 @@ func TestToBytes(t *testing.T) {
 	d.SetTransactionID(0xabcdef)
 	opt := OptionGeneric{OptionCode: 0, OptionData: []byte{}}
 	d.AddOption(&opt)
-	toBytes := d.ToBytes()
+	bytes := d.ToBytes()
```

Comment:
here I avoided the name `bytes` because it's the same of a standard library import, and if at some point we import it, this assignment will mask it in this function. Not a big deal but still to keep in mind

Reply:
Oh right, totally forgot about that. Good catch!